doc/source/configuration/shared-file-systems/drivers/vastdata_driver.rst

```diff
@@ -2,11 +2,11 @@
 Vastdata Share Driver
```

Comment:
VAST Share Driver

need to change VASTData -> VAST everywhere. The company name is VAST Data, with a space. But we only use 'VAST' in product names. 

Also, the driver name must be the same everywhere.